packages/angular_devkit/architect/BUILD.bazel

```diff
@@ -88,24 +90,25 @@ jasmine_node_test(
     srcs = [":architect_test_lib"],
 )
 
-# @external_begin
```

Comment:
Guessing we to keep the @external_begin & end here as well to not break google3 when this is synced in

Reply:
Nevermind. Not needed as per https://github.com/angular/angular-cli/pull/22258#discussion_r759582051